docs/en/serverless/quickstarts/auto-detect-logs-metrics.mdx

```diff
@@ -0,0 +1,96 @@
+---
+slug: /serverless/observability/quickstarts/auto-detect-logs-metrics
+title: Auto-detect logs and metrics
+description: Learn how to scan your hosts to detect and collect logs and metrics.
+tags: [ 'serverless', 'observability', 'how-to' ]
+---
+
+<p><DocBadge template="technical preview" /></p>
+
+In this quickstart guide, you'll learn how to scan your host to detect and collect logs and metrics,
+then navigate to dashboards to further analyze and explore your observability data.
+You'll also learn how to get value out of your observability data.
+
+## Prerequisites and limitations
+
+This quickstart has the following requirements and limitations:
+
+- The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to <DocLink slug="/serverless/general/assign-user-roles" />.
+- Root privileges on the host are required to run the auto-detection script used in this quickstart.
+- The auto-detection script currently scans for metrics and logs from Apache, Docker, Nginx, and the host system.
+   It also scans for custom log files.
+- The auto-detection script works on Linux and MacOS only and requires support for the `lsof` command.
```

Comment:
https://github.com/elastic/kibana/pull/189134 includes a fix so `lsof` is not a hard dependency. No need to mention this in the docs.

Reply:
I've modified the text. Let me know if my change works.